### ISSUE TITLE
Spell costs are ?!

### ISSUE THREAD

**Issue by catwiesel** [ZhengPeiRu21/mod-individual-progression#222]
I've started with azerothcore-wotlk and mod-i...-p...

I've noticed an issue with spell costs... they are higher than they should be, and the tooltip does not align with the actual costs.
I've dropped and repopulated the acore-world DB with different builds, and the issue seems to start when the .sql files of this mod are being run when populating the world sql table

I dont mind the higher costs. for now. but I do mind that the spell is shown as castable (not red) and you get a mana too low warning.

further number. new human priest, always lesser heal rank 1, total mana of the character, tooltip cost shown, actual taken

online wow server with progression, 123 mana, cost 30 mana, takes 30 mana
real classic wotlk, 160 mana, costs 17 mana, takes 17 mana
my server wotlk with or without modules while buidling, 123 mana, cost 11 mana, takes 11
my server with ( i believe your sql files being run against the db), 160 mana, cost 17 mana, takes 39 

after the issue has cropped up, I need to drop the db and create a new acore_world, switching chars, or running  with or without modules doe snot matter. which leads me to believe its a change made in sql, not in the actual build.

Please advise

**Comment by catwiesel** [comment]
oh and I need to add. this is not "this spell" or "this class" issue. Ive noticed this on my warlock, I just used a level 1 priest to check different servers/sql worlds/builds

**Comment by ValeTheVioletMote** [comment]
This is intentional, though poorly documented. If you've run the spell 
DB patch, then you need to have the optional patch in your client data 
(patch-V.MPQ).

The spell patch and the optional profession patch are locked together in 
the client patch, patch-V.

And to be clear, I don't think it should be this way, because it defeats 
the purpose of the profession patch being optional, and it causes this 
confusion because you wouldn't think to check an optional folder for a 
REQUIRED patch, but alas, you must.

I tried bringing it up to Zheng in the ACore discord but I don't think 
he understood what I meant.

**Comment by ValeTheVioletMote** [comment]
The solution ought to be that Zheng creates two patches: one with just 
the mana changes, one with just the profession changes, and one with 
both (which we already have, patch-V). Then document the mana spell only 
patch as required.

**Comment by ZhengPeiRu21** [comment]
> The solution ought to be that Zheng creates two patches: one with just the mana changes, one with just the profession changes, and one with both (which we already have, patch-V). Then document the mana spell only patch as required.

Agreed. DBCs are tricky since it isn't feasible to modularize the changes, I think having two variations in this case would be doable since it is so divisive. I'll add that in a future update.

**Comment by soniquete0** [comment]
Any update on this? I tried editing the provided DBC file to return original wotlk mana costs but sadly, due to it being an already edited file, no editor that I used (Spell Editor, WDBXeditor) managed to get the strings from the spell.dbc file, causing all the spell names and descriptions to be "Unknown Spell #".

**Comment by Grimfeather** [comment]
this file can now be edited

making multiple versions of spell.dbc only complicates things
use either the original file, or the edited file provided in dbc.7z
